src/scripts/Save.ts

```diff
@@ -23,12 +23,7 @@ class Save {
         let element = document.createElement('a');
         element.setAttribute('href', 'data:text/plain;charset=utf-8,' + encodeURIComponent(btoa(JSON.stringify(player))));
         let currentdate = new Date();
-        let datetime = "" + currentdate.getDate() + "/"
-            + (currentdate.getMonth() + 1) + "/"
-            + currentdate.getFullYear() + " @ "
-            + currentdate.getHours() + ":"
-            + currentdate.getMinutes();
-        let filename = "Pokeclicker save - " + datetime + '.txt';
+        let filename = "PokeClickerSave_" + currentdate.toISOString() + '.txt';
```

Comment:
```suggestion
        let filename = 'PokeClickerSave_' + currentdate.toISOString().replace(/T|\.\d+Z/g, ' ').replace(/:/g, '-').trim() + '.txt';
```
Output: `2019-05-29 22-49-53`